platform/darwin/src/MGLMetricsManager.m

```diff
@@ -0,0 +1,57 @@
+#import "MGLMetricsManager_Private.h"
+#import "MGLNetworkConfiguration_Private.h"
+#if TARGET_OS_IOS
+#import "MGLMapboxEvents.h"
+#endif
+
+NSString* MGLStringFromMetricType(MGLMetricType metricType) {
+    return [@[kMGLDownloadPerformanceEvent] objectAtIndex:metricType];
+}
+
+@interface MGLMetricsManager() <MGLNetworkConfigurationMetricsDelegate>
+
+@end
+
+@implementation MGLMetricsManager
+
++ (instancetype)sharedManager
+{
+    static dispatch_once_t once;
+    static id sharedConfiguration;
+    dispatch_once(&once, ^{
+        sharedConfiguration = [[self alloc] init];
+        [MGLNetworkConfiguration sharedManager].metricsDelegate = sharedConfiguration;
```

Comment:
Ideally we'd try to remove this explicit dependency - but let's think about that as follow on work.

Reply:
Sure. Just a remainder that `MGLNetworkConfiguration` is part of another target and there should not be explicit dependencies.